### PR TITLE
 feat: add RadioGroup component 

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/common/components/radioGroup/radioItem/RadioItem.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  labelTitle: string
+  disabled: boolean
+}
+export const RadioElement = ({ value, labelTitle, disabled }: RadioItem): JSX.Element => {
+  return (
+    <div className={s.itemContainer}>
+      <Typography
+        as={'label'}
+        variant={TypographyVariant.Body2}
+        className={cn(s.label, disabled && s.disabled)}
```
Comment (vitalyabedik) [inline src/common/components/radioGroup/radioItem/RadioItem.tsx:21]: Attribute className should be the first, higher only attribute ref

**Review by vitalyabedik — APPROVED** [review]

**Review by DenisKulik — APPROVED** [review]
good job :)

File: src/common/components/radioGroup/radioItem/RadioItem.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export type RadioItem = {
+  value: string
+  labelTitle: string
+  disabled: boolean
+}
+export const RadioElement = ({ value, labelTitle, disabled }: RadioItem): JSX.Element => {
+  return (
+    <div className={s.itemContainer}>
+      <Typography
+        as={'label'}
```
Comment (DenisKulik) [inline src/common/components/radioGroup/radioItem/RadioItem.tsx:19]: don't use curly braces when passing a string as a value (bad practice)

**Review by DenisKulik — CHANGES_REQUESTED** [review]

**Review by DenisKulik — APPROVED** [review]

File: src/common/components/radioGroup/radioItem/RadioItem.tsx (hunk 1 above)
Comment (VitaliKozhurov) [reply src/common/components/radioGroup/radioItem/RadioItem.tsx:19]: Done!

**Review by vitalyabedik — APPROVED** [review]
Well done!

File: src/common/components/radioGroup/radioItem/RadioItem.tsx (hunk 2 above)
Comment (VitaliKozhurov) [reply src/common/components/radioGroup/radioItem/RadioItem.tsx]: Done, delete curly braces!

**Pull request merged**